src/SimpleThings/EntityAudit/AuditReader.php

```diff
@@ -748,12 +748,24 @@ public function getEntityValues($className, $entity)
     {
         $metadata = $this->em->getClassMetadata($className);
         $fields = $metadata->getFieldNames();
+        $associationsFields = $metadata->getAssociationNames();
 
         $return = array();
         foreach ($fields AS $fieldName) {
             $return[$fieldName] = $metadata->getFieldValue($entity, $fieldName);
         }
 
+        foreach($associationsFields as $key => $association) {
+            $associationValues = $entity->{'get' . ucfirst($association)}();
```

Comment:
This is not reliable.

A getter for `available` property can be `isAvailable` for example.

You should use the [Symfony PropertyAccess component](https://symfony.com/doc/current/components/property_access.html).